package.json

```diff
@@ -1,77 +1,78 @@
-		"regenerator-runtime": "^0.13.3",
-		"stylelint": "^13.3.0",
-		"stylelint-config-recommended": "^3.0.0",
-		"stylelint-config-styled-components": "^0.1.1",
-		"stylelint-processor-styled-components": "^1.10.0",
-		"typescript": "^3.7.2",
-		"webpack": "^4.41.5"
-	},
-	"jest": {
-		"testPathIgnorePatterns": [
-			"<rootDir>/node_modules",
-			"<rootDir>/dist"
-		]
-	}
+  "name": "source",
+  "version": "0.18.0-rc.0",
+  "scripts": {
+    "storybook": "yarn watch:foundations & yarn watch:svgs & start-storybook",
+    "lint:styles": "stylelint src/core/**/styles.ts",
+    "lint:js": "eslint . --ext .ts,.tsx,.js",
+    "lint": "yarn lint:js && yarn lint:styles",
+    "fix": "yarn lint:js --fix",
+    "tsc": "tsc",
+    "test": "jest",
+    "validate": "yarn tsc && yarn lint && yarn test",
+    "clean": "rm -rf dist",
+    "build:storybook": "build-storybook -o dist",
+    "watch:foundations": "cd src/core/foundations && yarn watch",
+    "watch:svgs": "cd src/core/svgs && yarn watch",
+    "clean:all": "node ./scripts/clean-all",
+    "build:all": "node ./scripts/build-all",
+    "ci:build": "NODE_ENV=production yarn build:all && yarn build:storybook",
+    "publish:release": "node ./scripts/publish-release",
+    "verbump:minor": "yarn version --minor --no-git-tag-version",
+    "verbump:preminor": "yarn version --preminor --preid rc --no-git-tag-version",
+    "verbump:patch": "yarn version --patch --no-git-tag-version",
+    "verbump:prerelease": "yarn version --prerelease --preid rc --no-git-tag-version",
+    "verbump:preminor:all": "node ./scripts/verbump-preminor-all",
+    "verbump:minor:all": "node ./scripts/verbump-minor-all",
+    "verbump:patch:all": "node ./scripts/verbump-patch-all",
+    "verbump:prerelease:all": "node ./scripts/verbump-prerelease-all"
+  },
+  "private": true,
+  "workspaces": [
+    "src/core/*",
+    "src/core/components/*",
+    "src/editorial/web/components/*"
+  ],
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
+    "@babel/preset-env": "^7.9.0",
+    "@babel/preset-typescript": "^7.9.0",
+    "@emotion/babel-preset-css-prop": "^10.0.14",
+    "@emotion/core": "^10.0.27",
+    "@storybook/addon-backgrounds": "^5.3.18",
+    "@storybook/addon-viewport": "^5.3.18",
+    "@storybook/react": "^5.3.18",
+    "@types/jest": "^25.2.1",
+    "@types/react": "^16.9.2",
+    "@types/storybook__react": "^4.0.2",
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
+    "babel-jest": "^25.2.6",
+    "babel-loader": "^8.0.6",
+    "emotion-theming": "^10.0.27",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.1.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-react": "^7.14.3",
+    "execa": "^4.0.0",
+    "jest": "^25.2.7",
+    "prettier": "^1.18.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "regenerator-runtime": "^0.13.3",
+    "storybook-chromatic": "^3.5.2",
```

Comment:
It looks like this line is the substantive change. Do you know why Yarn has replaced all those lovely tabs with spaces?

Reply:
Because I ran an `npx` command which changed some stuff and the `.editorconfig` file doesn't specify tabs for the `package.json` file so it defaulted to spaces

I fixed it 😉

Reply:
Right. I figured it would inherit from [`[*]`](https://github.com/guardian/source/blob/master/.editorconfig#L8) but maybe I need to be more explicit?